carta/image.py

```diff
@@ -593,6 +593,83 @@ def hide_contours(self):
         """Hide the contours."""
         self.set_contours_visible(False)
 
+    # VECTOR OVERLAY
+
+    @validate(Constant(VectorOverlaySource), Constant(VectorOverlaySource), NoneOr(Number()), Boolean(), NoneOr(Number()), NoneOr(Number()), NoneOr(Number()))
+    def configure_vector_overlay(self, angular_source=VectorOverlaySource.CURRENT, intensity_source=VectorOverlaySource.CURRENT, pixel_averaging=4, fractional_intensity=False, threshold=None, q_error=None, u_error=None):
+        """Configure vector overlay.
+
+        Parameters
+        ----------
+        angular_source : {0}
+            The angular source. By default the current image is used.
+        intensity_source : {1}
+            The intensity source. By default the current image is used.
+        pixel_averaging : {2}
+            The pixel averaging width in pixel. Set to ``None`` to disable pixel averaging.
+        fractional_intensity : {3}
+            Enable fractional polarization intensity. By default the absolute polarization intensity is used.
+        threshold : {4}
+            The threshold in Jy/pixel. By default the threshold is disabled.
+        q_error : {5}
+            The Stokes Q error in Jy/beam. Set both this and ``u_error`` to enable debiasing. The debiasing is disabled by default.
+        u_error : {6}
+            The Stokes U error in Jy/beam. Set both this and ``q_error`` to enable debiasing. The debiasing is disabled by default.
+        """
+        pixel_averaging_enabled = pixel_averaging is not None
+        threshold_enabled = threshold is not None
+        if q_error is not None and u_error is not None:
+            debiasing = True
+        elif q_error is None and u_error is None:
+            debiasing = False
+        else:
+            print("q_error and u_error must be both set to enable debiasing.")
```

Comment:
Please use the logger for this message (you'll have to import it at the top; it's in `.util` -- see how it's used in e.g. `session.py`).

You could compact this using syntax similar to the previous lines (as shown below), but it does mean restating part of the same conditional, so I don't feel that strongly about it -- maybe the way it is now is clearer to read.

```
debiasing = q_error is not None and u_error is not None
if not debiasing and (q_error is not None or u_error is not None):
    logger.warning(...)
```

In the message, please swap "be both" -> "both be". I would also refer to the error variables by their descriptions rather than their variable names here ("The Stokes Q error and the Stokes U error must...")